packages/app-bridge/src/tests/AppBridgeThemeStub.ts

```diff
@@ -3,10 +3,6 @@
 import mitt, { Emitter } from 'mitt';
 import { SinonStubbedInstance, spy, stub } from 'sinon';
 
-import { EmitterEvents } from '../types/Emitter';
-import { AppBridgeTheme } from '../AppBridgeTheme';
-import { mergeDeep } from '../utilities';
```

Comment:
Reduce noise, please set up your ide to prevent that or revert the changes

Reply:
Reverted